lib/onc_certification_g10_test_kit/bulk_export_validation_tester.rb

```diff
@@ -184,6 +194,19 @@ def check_file_request(url) # rubocop:disable Metrics/CyclomaticComplexity
       line_count
     end
 
+    def process_validation_errors(total)
+      return if @invalid_line_count.zero?
+
+      first_error_message = "The line number for the first failed resource is #{first_error[:line_number]}."
+
+      messages.clear
+      messages.concat(first_error[:messages])
```

Comment:
This seems unnecessary given the assertion below.

Reply:
This is to keep errors from the first failed resource and pass to the test_result and repository later

Reply:
Oh, right. I was mixing up `first_error[:messages]` and `first_error_message`.